md/runtime-monitoring.md

```diff
@@ -0,0 +1,102 @@
+# Bonita Platform Monitoring
+
+Discover how to monitor a runtime environment running Bonita Platform 
+
+## Why monitoring ?
+
+Monitoring a Production environment is crucial to ensure the platform is correctly sized and tuned.
+
+Bonita provides a series of technical and BPM-related metrics to monitor the health of Bonita platform.  
+Some metrics are enabled by default and cannot be disabled. Some others are optional and can be enabled according to
+your needs.
+
+## Glossary
+
+**Work**: a unit piece of code that executes processes, tasks, BPM elements... and allows the processes to execute forwards.
+It executes in a Java thread.
+
+**Connector work**: a unit piece of code that specifically executes Bonita connectors
+
+**Work queue**: a queue storing pending works before they are taken by threads for execution 
```

Comment:
_note: this question is probably not really related to the contribution, but i'm interested :p _

So i guess that executing a work can lead to the creation of new work ? (executing a process create work to execute BPM elements?) How does the queue manage the priority? I would say if a work create subworks it create its own queue something like that ?

Reply:
There is no notion of subworks, a work triggers 1 other work if necessary, once it has finished its business.
The queue is FIFO, so there is no derogation / priority in the current implementation.

Reply:
OK so I guess that when you say _a unit piece of code that executes processes_, it doesn't mean that a work is responsible of the whole execution of a process.

Reply:
Right. I should reformulate. Do you suggest a better sentence?

Reply:
*a unit piece of code that executes parts of process instances* is better to you?

Reply:
Well since I'm not exactly sure of what it does no ^^'